org.sonarlint.eclipse.cdt/META-INF/MANIFEST.MF

```diff
@@ -14,7 +14,9 @@ Require-Bundle: org.eclipse.core.runtime,
  org.sonarlint.eclipse.core,
  org.eclipse.core.filesystem,
  org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.text
+ org.eclipse.text,
```

Comment:
what do these lines do? just curious

Reply:
It is basically the Eclipse way of defining dependencies at runtime (and therefore also compile time).

The `MANIFEST.MF` and specifically this section will contain all bundles that we require at the runtime of the Eclipse installation. Most of them are present due to them being part of the Eclipse platform itself, but for some we have an optional.

E.g. here we have the `org.eclipse.cdt.core;resolution:=optional` statement that more or less is used for us to only make use of this very sub-plugin if CDT (the C development tools) are installed.

What I added is the compare bundle that brings in the `TextMergeViewer` and SWT that is the UI system of Eclipse.